taskcluster/kinds/integration-image/kind.yml

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - fxci_config_taskgraph.transforms.integration_image
+  - taskgraph.transforms.run
+  - taskgraph.transforms.task
+
+task-defaults:
+  worker-type: t-linux
+  worker:
+    docker-image: {in-tree: python3.11}
+    env:
+      MOZ_ARTIFACT_DIR: /builds/worker/artifacts
+    max-run-time: 3600
+  run:
+    using: run-task
+    cwd: '{checkout}'
+    command: >-
+      curl -o $MOZ_ARTIFACT_DIR $FETCH_URL
```

Comment:
`--fail --location --retry 5` (assuming we can't / don't want to use fetch-content here...)

Reply:
Thanks, this was a placeholder while I was working out the dependencies. But you saved me some Googling, so appreciated! :)

The problem with `fetch-content` was that it requires knowing the hash and size, which I guess we could find a way to obtain if we were motivated enough.. but imo meh.

Reply:
Although I guess I could do the same use `task-artifact` fetches and fake the TASKCLUSTER_ROOT_URL trick I'm doing with the test tasks...